cookbooks/aws-parallelcluster-common/resources/lustre/lustre_redhat8.rb

```diff
@@ -39,8 +39,14 @@
 end
 
 action_class do
+  def kernel_minor_patch
+    node['cluster']['kernel_release'].match(/^\d+\.\d+\.\d+-(\d+)\..*$/)[1]
```

Comment:
why not using same approach done for centos7?
This change:
* requires further modification once new minor will be out, while the centos7 code only requires a new line to be added
* doesn't cover the case of minor version not supported, e.g. 8.3

Reply:
> requires further modification once new minor will be out, while the centos7 code only requires a new line to be added
> doesn't cover the case of minor version not supported, e.g. 8.3

Since we are not pinning any minor version, the recommended approach from Fsx is to point to the plain 8 repo, but at the moment the correct package is only in the 8.7 repo and the other minor version repo does not exist.
Also with centos7 we would need to add a condition if a new minor is released. In any case, however, if a new minor not compatible with the 8.7 is released, I would expect the installation to fail.

Reply:
Refactored and followed the step in the [public doc](https://docs.aws.amazon.com/fsx/latest/LustreGuide/install-lustre-client.html#lustre-client-rhel)
Thanks